samples/opencl-c-interop/opencl-c-interop.cpp

```diff
@@ -33,6 +33,8 @@
 
 using namespace cl::sycl;
 
+class pow_comp;
```

Comment:
This one hasn't removed the `class` from the call site, not an error but would be consistent